partner_sale_risk/models/res_partner.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    risk_sale_order_include = fields.Boolean(
+        string='Include Sales Orders', help='Full risk computation')
+    risk_sale_order_limit = fields.Monetary(
+        string='Limit Sales Orders', help='Set 0 if it is not locked')
+    risk_sale_order = fields.Monetary(
```

Comment:
Default currency_id field name is 'currency_id'

https://github.com/odoo/odoo/blob/aa09c522053b8c91dea557f9e9e71be2f4e965be/addons/account/models/partner.py#L414

https://github.com/odoo/odoo/blob/bbdf9605e4c480e1fc19b1faac9cfc4321bcaea2/openerp/fields.py#L1162-L1163